nsqdserver/context.go

```diff
@@ -175,6 +175,25 @@ func (c *context) DeleteExistingChannel(topic *nsqd.Topic, channelName string) e
 	return c.nsqdCoord.DeleteChannel(topic, channelName)
 }
 
+func (c *context) UpdateChannelState(ch *nsqd.Channel, paused int, skipped int) error {
+	var err error
+	if c.nsqdCoord == nil {
+		switch paused {
+		case 1:
+			err = ch.Pause()
+		case 0:
+			err = ch.UnPause()
```

Comment:
switch for skip added in https://github.com/absolute8511/nsq/pull/32/commits/3c68264fd036d0d6829f69449db871e225532122